sej-videos.html

```diff
@@ -22,6 +22,29 @@ <h1>Inspirational Videos</h1>
 
  </div>
 </section>
+
+  <div id="poster">
```

Comment:
ID should be unique so there is only one per page. You can have a class on multiple elements